protocol/avdl/keybase1/prove_ui.avdl

```diff
@@ -53,6 +53,11 @@ protocol proveUi {
   // Used by parameterized proofs only
   void checking(int sessionID, string name);
 
+  // The proof is being polled, ask whether to continue checking.
+  // Returns whether to keep checking.
+  // Used by parametrized proofs only.
+  boolean continueChecking(int sessionID, string name);
```

Comment:
`string name` look unused

Reply:
yep! do you think passing in `0` as sessionID in `engine/prove` is better?

Reply:
yes and better yet you can, in engine/prove, call `m.UIs().ProveUI.ContinueChecking(m.Ctx())`. And change `ContinueChecking` in service and interfaces to take no arg.